### ISSUE TITLE
Set "additionalProperties" to true for everything

### ISSUE THREAD

**Issue by evanplaice** [jsonresume/resume-schema#373]
If users want to customize the schema, I say let them

This 'official' version of a schema exists so tooling/template implementors have a sane target but if users want to extend the hell out of it and design their own template format, lets enable them.

**Comment by thomasdavis** [comment]
Labelled as RFC.

It's hard to balance a schema with implementation. 

Given theme developers the luxury to deviate, evidently impairs the practicality of schematic consensus.

**Comment by treyhuffine** [comment]
I agree with @evanplaice's sentiment. JSON Resume provides a standard for the community, but when you're building something that is meant for your own use case (which can be very often for a resume), it would be nice to not feel like you have to fight a tool to get the result you want.

Another option to enhance it could be a strict mode where you are forced to follow the schema exactly, or you can toggle it off to give the freedom to extend/build what suits a personal use case if the result is not meant to be an "official" part of the ecosystem.

Removing constraints and giving people more freedom could also generate some interesting and powerful extensions that otherwise may have never been tried. As long as the official schema still exists to provide an anchor for the devs building tooling and templates, I think this would ultimately be a net positive for everyone.

**Comment by thomasdavis** [comment]
Good news on a 2020 RFC: this is how the schema works on master today. Every object in `schema.json` now sets `additionalProperties: true` — the root, `meta`, and every section's items — so custom fields validate fine anywhere, and the official schema stays a sane anchor for tooling rather than a straitjacket. Closing as resolved.

For context: JSON Resume is under active maintenance again (2026) and the project has consolidated into the jsonresume.org monorepo — the schema now lives there as `packages/schema` (the `@jsonresume/schema` npm package is unchanged). See jsonresume/jsonresume.org#275. If you'd like to push extensibility further (e.g. formal extension points), a fresh issue on the monorepo is very welcome.
